TileStache/Mapnik.py

```diff
@@ -308,7 +330,7 @@ def renderArea(self, width, height, srs, xmin, ymin, xmax, ymax, zoom):
             finally:
                 global_mapnik_lock.release()
 
-        logging.debug('TileStache.Mapnik.GridProvider.renderArea() %dx%d at %d in %.3f from %s', width, height, self.scale, time() - start_time, self.mapfile)
+        logging.debug('TileStache.Mapnik.GridProvider.renderArea() %dx%d at %d in %.3f from %s', width, height, self.scale, time() - start_time, self.mapconfig)
```

Comment:
I am thinking that for debugging purposes, you do want to see the xml config. Or how else you would see it?